seeds/userSeeds.json

```diff
@@ -1,15 +1,42 @@
 [
-    {
-      "username": "JamesMurray",
-      "password": "password12345"
-    },
-    {
-      "username": "Salaster",
-      "password": "password12345"
-    },
-    {
-      "username": "JoeGatto",
-      "password": "password12345"
-    }
+  {
+    "username": "JamesMurray",
+    "password": "password12345"
+  },
+  {
+    "username": "Sal Vulacano",
+    "password": "password12345"
+  },
+  {
+    "username": "Joe Gatto",
+    "password": "password12345"
+  },
+  {
+    "username": "Theo Von",
+    "password": "password12345"
+  },
+  {
+    "username": "Micheal Carbanaro",
+    "password": "password12345"
+  },
+  {
+    "username": "JoeGatto",
+    "password": "password12345"
+  },
+  {
+    "username": "JoeGatto",
+    "password": "password12345"
+  },
+  {
+    "username": "JoeGatto",
+    "password": "password12345"
+  },
+  {
+    "username": "JoeGatto",
+    "password": "password12345"
+  },
+  {
+    "username": "JoeGatto",
+    "password": "password12345"
+  }
 ]
-  
```

Comment:
I think this is a JSON syntax thing too--it's complaining that there isn't a blank line at the end...